packages/fether-react/src/Tokens/Tokens.js

```diff
@@ -56,15 +93,44 @@ class Tokens extends PureComponent {
       menuItems.unshift(backupAccountItem);
     }
 
+    if (showWarning) {
+      menuItems.push(backupRecoveryItem);
+    }
+
     return menuItems;
   };
 
+  menuTrigger = () => {
+    const { showWarning } = this.state;
+
+    return (
+      <span style={{ display: 'flex' }}>
+        <Clickable className='icon -menu' />
+        {showWarning && <a className='icon -warning' />}
```

Comment:
This makes the warning appear as a separate button next to the warning, it looks weird on mouse over. I managed to get a good look and feel ([preview](https://user-images.githubusercontent.com/33178835/53008451-bb336a80-3439-11e9-9a91-cbafe288dfe6.png)) with a `span` and the following styling, if that can help:
```css
    background-image: url(/static/media/warning.7fd1457c.svg);
    width: 1.8rem;
    background-size: .7rem auto;
    background-position: center center;
    background-repeat: no-repeat;
    margin-left: -.5rem;
    margin-bottom: .6rem;
```